Signal/src/Profiles/ProfileFetcherJob.swift

```diff
@@ -105,7 +114,10 @@ class ProfileFetcherJob: NSObject {
     private func updateProfile(signalServiceProfile: SignalServiceProfile) {
         verifyIdentityUpToDateAsync(recipientId: signalServiceProfile.recipientId, latestIdentityKey: signalServiceProfile.identityKey)
 
-        // Eventually we'll want to do more things with new SignalServiceProfile fields here.
+        OWSProfileManager.shared().updateProfile(forRecipientId : signalServiceProfile.recipientId,
```

Comment:
Yeah, I thought about it but:

a) Didn't yet because this is already working.
b) This is swift and the manager is obj-c (since it was born in SSK).
c) Throttling is all done in one place - but we sometimes (may) want to make an exception to it.

If you want to migrate it, +1.